homeassistant/components/wled/const.py

```diff
@@ -9,8 +9,8 @@
 SCAN_INTERVAL = timedelta(seconds=10)
 
 # Options
-CONF_KEEP_MASTER_LIGHT = "keep_master_light"
-DEFAULT_KEEP_MASTER_LIGHT = False
+CONF_KEEP_MAIN_LIGHT = "keep_master_light"
```

Comment:
```suggestion
CONF_KEEP_MAIN_LIGHT = "keep_main_light"
```

Reply:
This would require a migration, how far do we want to pull this?

Reply:
Right, let's not do that right now 👍